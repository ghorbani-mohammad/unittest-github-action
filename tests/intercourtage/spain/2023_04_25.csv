

Martes 25 Abril 2023,,,,,,

"Puerto: Tarragona","","Estado de la llegada: Descargando","","",""


"Producto","Toneladas","Origen","Importador","Consignatario/Estiba","Barco"
"MAIZ","44.000","UCRANIA","BUNGE IBERICA","AROLA EUROPORTS","AKSON SERIN"


,,,Estado de la llegada: En previsión,,,

"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"DDG","16.100","EUA","CEFETRA IBERICA","ALFASHIP ERSHIP","DISCOVERY"
"GUISANTES","15,400","MAR NEGRO","RIERA ROURA","MARMEDSA NOATUM CODEMAR","REK KING"
"MAIZ","35.000","MAR NEGRO","RIERA ROURA","","SUMMER LADY"
"TRIGO","30.000","MAR NEGRO","RIERA ROURA","","SUMMER LADY"
"TRIGO","30.000","MAR NEGRO","RIERA ROURA","","BARBRO G"
"MAIZ","29,600","UCRANIA","ETG COMMODITIES","ERSHIP ERSHIP","GULMAR"
"HARINA DE GIRASOL","4.900","UCRANIA","VITERRA AGRICOLA ESPANA","ERSHIP ERSHIP","NIKITI II"
"TRIGO","46.000","UCRANIA","CARGILL","ALFASHIP ERSHIP","POPI S"
"TRIGO PANIFICABLE","51.700","CANADÁ","CEGECO","ERSHIP ERSHIP","PORT KOBE"


"Puerto: Barcelona","","","Estado de la llegada: Descargando","",""


"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"HABAS DE SOJA","58.000","BRASIL","CARGILL","MASIQUES CARGILL","OMICRON SKY"


"Puerto: Valencia","","","Estado de la llegada: Descargando","",""


"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"","","","","",""
"","","","","",""


"TRIGO","33.000","RUMANIA","CEFETRA IBERICA","ERSHIP TEMAGRA","CARIBBEAN SPIRIT"


"","","","Estado de la llegada: En previsión","-",""


"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"CEBADA","13.500","ALEMANIA","CEFETRA IBERICA","ERSHIP TEMAGRA","VIPHA NAREE"
"HARINA DE SOJA","15.000","BRASIL","COFCO","ERSHIP TEMAGRA","RED ORCHID"
"MAIZ","43.000","UCRANIA","CEFETRA IBERICA","ERSHIP TEMAGRA","LUCENT"


Puerto: Castellón,,,,,,

,,,Estado de la llegada: Descargando,,,

"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"TRIGO PANIFICABLE","42.500","RUMANIA","CARGILL","PORTSUR","S RUMBA"


,,,Estado de la llegada: En previsión,,,

"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"TRIGO PANIFICABLE","6.500","BULGARIA","HARINERA DELMAR","PORTSUR","NAVIN FALCON"

