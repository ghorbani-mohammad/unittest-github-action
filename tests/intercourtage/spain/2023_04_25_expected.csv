report_date,contributor_lineup_reference_id,commodity_name,commodity_specification_codes,volume,unit,origin_code,destination_code,expected_cargo_loading_date,cargo_loading_date,cargo_discharged_date,cargo_status,carrier_id,carrier_name,carrier_status,carrier_dead_weight,export_location,import_location,export_terminal,import_terminal,eta,etb,etd,ata,atb,atd,agent,charterer,shipper,operator,comments
None,None,MAIZ,None,44000.0,MT,None,None,None,None,2023-04-25T00:00:00+00:00,None,None,AKSON SERIN,None,None,UCRANIA,Tarragona,None,None,None,None,None,None,None,None,None,BUNGE IBERICA,None,AROLA EUROPORTS,None
None,None,HABAS DE SOJA,None,58000.0,MT,None,None,None,None,2023-04-25T00:00:00+00:00,None,None,OMICRON SKY,None,None,BRASIL,Barcelona,None,None,None,None,None,None,None,None,None,CARGILL,None,MASIQUES CARGILL,None
None,None,TRIGO,None,33000.0,MT,None,None,None,None,2023-04-25T00:00:00+00:00,None,None,CARIBBEAN SPIRIT,None,None,RUMANIA,Valencia,None,None,None,None,None,None,None,None,None,CEFETRA IBERICA,None,ERSHIP TEMAGRA,None
None,None,TRIGO PANIFICABLE,None,42500.0,MT,None,None,None,None,2023-04-25T00:00:00+00:00,None,None,S RUMBA,None,None,RUMANIA,Castell√≥n,None,None,None,None,None,None,None,None,None,CARGILL,None,PORTSUR,None
